mmv1/products/vertexai/FeatureOnlineStoreFeatureview.yaml

```diff
@@ -0,0 +1,127 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: FeatureOnlinestoreFeatureview
+base_url: projects/{{project}}/locations/{{region}}/featureOnlineStores/{{feature_online_store}}/featureViews
+create_url: projects/{{project}}/locations/{{region}}/featureOnlineStores/{{feature_online_store}}/featureViews?featureViewId={{name}}
+self_link: projects/{{project}}/locations/{{region}}/featureOnlineStores/{{feature_online_store}}/featureViews/{{name}}
+update_verb: :PATCH
+update_mask: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/vertex-ai/docs'
+  api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.featureOnlineStores.featureViews'
+async: !ruby/object:Api::OpAsync
+  actions:
+    - create
+    - delete
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+description: |-
+  FeatureView is representation of values that the FeatureOnlineStore will serve based on its syncConfig.
+import_format:
+  [
+    'projects/{{project}}/locations/{{region}}/featureOnlineStores/{{feature_online_store}}/featureViews/{{name}}',
+  ]
+autogen_async: false
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'vertex_ai_featureonlinestore_featureview'
+    primary_resource_id: 'featureview'
+    vars:
+      name: 'tf_test_fv1'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: featureOnlineStore
+    description:
+      The name of the FeatureOnlineStore to use for the featureview.
+    url_param_only: true
+    immutable: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'region'
+    description: The region for the resource. It should be the same as the featureonlinestore region.
+    required: true
+    immutable: true
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description:
+      Name of the FeatureView. This value may be up to 60 characters, and
+      valid characters are [a-z0-9_]. The first character cannot be a number.
+    immutable: true
+    url_param_only: true
+    pattern: 'projects/{{project}}/locations/{{region}}/featureOnlineStores/{{featureOnlineStore}}/featureViews/{{name}}'
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+  - !ruby/object:Api::Type::String
+    name: 'etag'
+    description: Used to perform consistent read-modify-write updates.
+    output: true
+    ignore_read: true
```

Comment:
ignore_read + output doesn't make sense. Just remove this

Reply:
This is still present

Reply:
@slevenick Without that, the example testcases are failing with `ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected` error for `eTag` attribute. Also I've seen that ignore_read: true is set for all the `vertexai` resources. Hence, I added it back.

Reply:
removed the eTag field as discussed.